### ISSUE TITLE
Does not compile on Linux due to missing modules

### ISSUE THREAD

**Issue by philpax** [EqualGames/game-scanner#22]
## Expected Behavior

`game-scanner` should build on Linux with no issues.

## Current Behavior

There are missing modules and other implementations (due to the use of cfg os-gating) that are not available on Linux:
```
error[E0583]: file not found for module `utils`
  --> /home/philpax/.cargo/registry/src/github.com-1ecc6299db9ec823/game-scanner-1.1.2/src/epicgames/mod.rs:14:1
   |
14 | mod utils;
   | ^^^^^^^^^^
   |
   = help: to create the module `utils`, create file "/home/philpax/.cargo/registry/src/github.com-1ecc6299db9ec823/game-scanner-1.1.2/src/epicgames/utils.rs" or "/home/philpax/.cargo/registry/src/github.com-1ecc6299db9ec823/game-scanner-1.1.2/src/epicgames/utils/mod.rs"

error[E0583]: file not found for module `utils`
  --> /home/philpax/.cargo/registry/src/github.com-1ecc6299db9ec823/game-scanner-1.1.2/src/gog/mod.rs:10:1
   |
10 | mod utils;
   | ^^^^^^^^^^
   |
   = help: to create the module `utils`, create file "/home/philpax/.cargo/registry/src/github.com-1ecc6299db9ec823/game-scanner-1.1.2/src/gog/utils.rs" or "/home/philpax/.cargo/registry/src/github.com-1ecc6299db9ec823/game-scanner-1.1.2/src/gog/utils/mod.rs"
```
(so on and so forth, there's more errors like this)

## Possible Solution

Implement the relevant utils for Linux, or disable these modules on Linux.

## Steps to Reproduce (for bugs)

Try to build `game-scanner` by itself, or as a dependency, on x86-64 Linux.

## Context

Was just testing my existing code on Linux and noticed this. Not at all an issue - going to be running in Wine anyway - but I figured it was worth calling out. For now, I'm only including game-scanner on Windows as a workaround. 

## Your Environment

* Version used: 1.1.2
* Operating System and version: Arch Linux, HEAD all packages

**Comment by matheusalbino** [comment]
@philpax fixed on v1.1.3

**Comment by philpax** [comment]
Awesome, nice work with both issues! 🎉
